### PR TITLE
Add github issue templates

### PR DESCRIPTION
**Pull request by seagemgames**
Can someone review these and make sure they fit here? I feel that the item repo's build error should specify that it's only for when an item causes a build error and not the compiler.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/build-error.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Build Error
+about: Report a building error
```
Comment (Thedoczek) [inline .github/ISSUE_TEMPLATE/build-error.md:3]: Change "Build Error" to "Compile Error"

**Review by Thedoczek — APPROVED** [review]
Kinda good

File: .github/ISSUE_TEMPLATE/build-error.md (hunk 1 above)
Comment (seagemgames) [reply .github/ISSUE_TEMPLATE/build-error.md:3]: I used "Build Error" to match the one in the item issues.

Comment (Luke18033) [reply .github/ISSUE_TEMPLATE/build-error.md:3]: And *I* used "Build Error" in the item template to match the in-game message, to make it more obvious what this is referring to.

**Comment by Thedoczek** [comment]
Ok

File: .github/ISSUE_TEMPLATE/build-error.md (hunk 1 above)
Comment (Thedoczek) [reply .github/ISSUE_TEMPLATE/build-error.md:3]: ok

**Pull request merged**